web/server/console.py

```diff
@@ -99,8 +99,7 @@ def log_test_results(
         self.queue.put_nowait(self._make_event(data, event="tests", ok=ok))
 
     def log_success(self, msg: str) -> None:
-        self.queue.put_nowait(self._make_event(msg))
-        self.stop_snapshot_progress()
+        self.queue.put_nowait(self._make_event(msg, event="promote-environment"))
```

Comment:
`log_success` can be called not just when we `promoting`, console also does `log_success` when virtual env is executed. So maybe not the best name for this event, but calling it just `success` may be too generic